sunpy/coordinates/transformations.py

```diff
@@ -57,7 +57,10 @@ def _carrington_offset(obstime):
     return get_sun_L0(obstime)
 
 
-def _observers_are_equal(obs_1, obs_2):
+def _observers_are_equal(obs_1, obs_2, string_ok=False):
+    if string_ok:
+        if obs_1 == obs_2:
```

Comment:
Shouldn't this just be `return obs_1 == obs_2`

Reply:
I didn't do that because if its false we need to run the rest of the function.

Reply:
Oh I see now. Yeah I think I did the same thing when I wrote the first version of this function...and then forgot!